.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
                   rustup install $(cat rust-toolchain)
                   rustup component add rustfmt-preview
-                  ./autogen.sh && ./configure --without-makeinfo --with-x=no --with-ns=no --without-gconf --without-gsettings
+                  ./autogen.sh && ./configure --without-makeinfo --with-x=no --with-ns=no --without-gconf --without-gsettings --with-native-compilation
```

Comment:
Should this be --without-native-compilation ?

Reply:
I thought that it might make sense to build linux with native-comp enabled. But I can turn it off until it also works for macOS.

Reply:
It was a little too late yesterday for me =)